### ISSUE TITLE
[Bug]: Moving persistent volumes to new location causes MongoDB to fail to start

### ISSUE THREAD

**Issue by Matthias-vdE** [mbentley/docker-omada-controller#463]
### Controller Version

v5.14.26.1

### Describe the Bug

I am currently running the container with volumes 

```
    volumes:
      - "omada-data:/opt/tplink/EAPController/data"
      - "omada-logs:/opt/tplink/EAPController/logs"
```

Because I'm migrating from docker run to docker compose, in preparation for migrating to new hardware, I have moved the volumes omada-data and omada-logs to omada-controller_omada-data and omada-controller_omada-logs.

To achieve this, I stop the running container, and on my host machine I perform a copy -a:

```
cp -afv /var/lib/docker/volumes/omada-data/. /var/lib/docker/volumes/omada-controller_omada-data/
cp -afv /var/lib/docker/volumes/omada-logs/. /var/lib/docker/volumes/omada-controller_omada-logs/
```

After doing so, and restarting the container pointing to the new location, mongodb fails to start.

### Expected Behavior

Since the data in both volumes is an exact copy, I expect mongoDB to start without issue.

### Steps to Reproduce

1. Stop the container.
2. Change the docker-compose file to point to the new persistent volumes
3. Start the container.

### How You're Launching the Container

```plain
Via Docker Compose:

services:
  omada-controller:
    container_name: "omada-controller"
    environment:
      - "MANAGE_HTTP_PORT=8088"
      - "MANAGE_HTTPS_PORT=8043"
      - "PGID=508"
      - "PORTAL_HTTP_PORT=8088"
      - "PORTAL_HTTPS_PORT=8843"
      - "PORT_ADOPT_V1=29812"
      - "PORT_APP_DISCOVERY=27001"
      - "PORT_DISCOVERY=29810"
      - "PORT_MANAGER_V1=29811"
      - "PORT_MANAGER_V2=29814"
      - "PORT_TRANSFER_V2=29815"
      - "PORT_RTTY=29816"
      - "PORT_UPGRADE_V1=29813"
      - "PUID=508"
      - "SHOW_SERVER_LOGS=true"
      - "SHOW_MONGODB_LOGS=false"
      - "TZ=Europe/Brussels"
    hostname: "omada-controller"
    image: "docker.io/mbentley/omada-controller:latest"
    network_mode: "bridge"
    ports:
      - "27001:27001/udp"
      - "29810:29810/udp"
      - "29811:29811/tcp"
      - "29812:29812/tcp"
      - "29813:29813/tcp"
      - "29814:29814/tcp"
      - "29815:29815/tcp"
      - "29816:29816/tcp"
      - "8043:8043/tcp"
      - "8088:8088/tcp"
      - "8843:8843/tcp"
    restart: "unless-stopped"
    volumes:
      - "omada-data:/opt/tplink/EAPController/data"
      - "omada-logs:/opt/tplink/EAPController/logs"

volumes:
  omada-data:
  omada-logs:


This is running on a Portainer instance, so the physical folders on the host machine are "omada-controller_omada-data" and "omada-controller_omada-logs".
```

### Container Logs

```plain
INFO: Validating user/group (omada:omada) exists with correct UID/GID (508:508)
INFO: Group (omada) doesn't exist; creating
INFO: User (omada) doesn't exist; creating
INFO: Time zone set to 'Europe/Brussels'
INFO: Value of 'manage.http.port' already set to 8088 in omada.properties
INFO: Value of 'manage.https.port' already set to 8043 in omada.properties
INFO: Value of 'portal.http.port' already set to 8088 in omada.properties
INFO: Value of 'portal.https.port' already set to 8843 in omada.properties
INFO: Value of 'port.adopt.v1' already set to 29812 in omada.properties
INFO: Value of 'port.app.discovery' already set to 27001 in omada.properties
INFO: Value of 'port.upgrade.v1' already set to 29813 in omada.properties
INFO: Value of 'port.manager.v1' already set to 29811 in omada.properties
INFO: Value of 'port.manager.v2' already set to 29814 in omada.properties
INFO: Value of 'port.discovery' already set to 29810 in omada.properties
INFO: Value of 'port.transfer.v2' already set to 29815 in omada.properties
INFO: Value of 'port.rtty' already set to 29816 in omada.properties
INFO: Value of 'mongo.external' already set to false in omada.properties
INFO: Value of 'eap.mongod.uri' already set to mongodb://127.0.0.1:27217/omada in omada.properties
WARN: Ownership not set correctly on '/opt/tplink/EAPController/properties'; setting correct ownership (omada:omada)
INFO: Version check passed; image version (5.14.26.1) >= the last version ran (5.14.26.1); writing image version to last ran file...
INFO: userland/kernel check passed
INFO: Starting Omada Controller as user omada
09-02-2024 13:59:59.457 INFO [main] [] c.t.s.o.s.OmadaLinuxMain(): record: start the omada controller
09-02-2024 13:59:59.463 INFO [main] [] c.t.s.o.s.OmadaLinuxMain(): record: set property finished
09-02-2024 13:59:59.499 INFO [main] [] c.t.s.o.s.OmadaLinuxMain(): record: configure log finished
09-02-2024 13:59:59.502 INFO [main] [] c.t.s.o.s.OmadaBootstrap(): record: bootstrap prepare
09-02-2024 13:59:59.532 INFO [main] [] c.t.s.o.c.o.a.b(): success to load configuration omada.properties
09-02-2024 13:59:59.534 INFO [main] [] c.t.s.o.c.o.OmadacType(): omadacType: Local Controller
09-02-2024 13:59:59.534 INFO [main] [] c.t.s.o.s.e.c(): Configuration omadacType is linux
09-02-2024 13:59:59.581 INFO [main] [] c.t.s.o.s.OmadaBootstrap(): going to start local mongod.
09-02-2024 14:01:39.712 ERROR [main] [] c.t.s.f.c.FacadeUtils(): facadeMsgEnable is not enable, msg: Fail to start mongo DB server
09-02-2024 14:01:39.712 WARN [main] [] c.t.s.o.s.s.b(): Fail to start mongo DB server
09-02-2024 14:01:39.713 WARN [main] [] c.t.s.o.s.s.a(): com.tplink.smb.omada.starter.b.a: Fail to start mongo DB server
com.tplink.smb.omada.starter.b.a: Fail to start mongo DB server
	at com.tplink.smb.omada.starter.server.b.a(SourceFile:156) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.server.a.e(SourceFile:74) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.d(SourceFile:262) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.q(SourceFile:345) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.a(SourceFile:125) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.e(SourceFile:292) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaLinuxMain.a(SourceFile:92) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaLinuxMain.main(SourceFile:42) ~[local-starter-5.14.26.1.jar:5.14.26.1]
09-02-2024 14:03:22.669 ERROR [main] [] c.t.s.f.c.FacadeUtils(): facadeMsgEnable is not enable, msg: Fail to start mongo DB server
09-02-2024 14:03:22.669 WARN [main] [] c.t.s.o.s.s.b(): Fail to start mongo DB server
09-02-2024 14:03:22.669 WARN [main] [] c.t.s.o.s.s.a(): com.tplink.smb.omada.starter.b.a: Fail to start mongo DB server
com.tplink.smb.omada.starter.b.a: Fail to start mongo DB server
	at com.tplink.smb.omada.starter.server.b.a(SourceFile:156) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.server.a.e(SourceFile:74) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.d(SourceFile:262) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.n(SourceFile:255) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.q(SourceFile:348) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.a(SourceFile:125) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.e(SourceFile:292) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaLinuxMain.a(SourceFile:92) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaLinuxMain.main(SourceFile:42) ~[local-starter-5.14.26.1.jar:5.14.26.1]
09-02-2024 14:03:36.674 INFO [mongo-check-thread] [] c.t.s.o.s.OmadaBootstrap(): mongod was shutdown, restarting it
09-02-2024 14:03:36.815 ERROR [mongo-check-thread] [] c.t.s.f.c.FacadeUtils(): facadeMsgEnable is not enable, msg: An unexpected error occurred in MongoDb. Omada Controller will try to restart it.
09-02-2024 14:03:53.090 WARN [main] [] c.t.s.o.s.OmadaBootstrap(): Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=127.0.0.1:27217, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused}}]
com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=127.0.0.1:27217, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused}}]
	at com.mongodb.internal.connection.BaseCluster.getDescription(BaseCluster.java:180) ~[mongodb-driver-core-4.6.1.jar:?]
	at com.mongodb.internal.connection.SingleServerCluster.getDescription(SingleServerCluster.java:44) ~[mongodb-driver-core-4.6.1.jar:?]
	at com.mongodb.client.internal.MongoClientDelegate.getConnectedClusterDescription(MongoClientDelegate.java:144) ~[mongodb-driver-sync-4.6.1.jar:?]
	at com.mongodb.client.internal.MongoClientDelegate.createClientSession(MongoClientDelegate.java:101) ~[mongodb-driver-sync-4.6.1.jar:?]
	at com.mongodb.client.internal.MongoClientDelegate$DelegateOperationExecutor.getClientSession(MongoClientDelegate.java:291) ~[mongodb-driver-sync-4.6.1.jar:?]
	at com.mongodb.client.internal.MongoClientDelegate$DelegateOperationExecutor.execute(MongoClientDelegate.java:183) ~[mongodb-driver-sync-4.6.1.jar:?]
	at com.mongodb.client.internal.FindIterableImpl.first(FindIterableImpl.java:213) ~[mongodb-driver-sync-4.6.1.jar:?]
	at com.tplink.smb.omada.starter.OmadaBootstrap.a(SourceFile:696) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.v(SourceFile:715) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.a(SourceFile:676) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaBootstrap.e(SourceFile:299) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaLinuxMain.a(SourceFile:92) ~[local-starter-5.14.26.1.jar:5.14.26.1]
	at com.tplink.smb.omada.starter.OmadaLinuxMain.main(SourceFile:42) ~[local-starter-5.14.26.1.jar:5.14.26.1]
```

### Additional Context

tl;dr: Copying the persistent volumes to a new location causes mongoDB to fail.

**Comment by Matthias-vdE** [comment]
Nevermind :) Deleting and re-creating the container seems to have fixed it.
